content/applications/sales/sales/send_quotations/deadline.rst

```diff
@@ -1,44 +1,57 @@
-===================================================
+==========================================
+Quotation deadlines to stimulate customers
+==========================================
 
-When sending quotations, it is important to set a deadline to encourage your customers to act.
-Indeed, this will stimulate them because they will be afraid of missing a good deal and it will also
-allow you to protect yourself in case you have to fulfill an order at a price that is no longer
-profitable for you.
+When sending quotations, it's important to set a deadline to encourage customers to act and confirm
+orders in a timely manner.
 
-Set a deadline
-==============
+Setting deadlines on quotations entices customers to finalize the process because they might fear
+that they'll miss out on a good deal. Quotation deadlines are also useful because they can be used
+as protection for a company, in case an order has to be fulfilled at a price that is no longer
+profitable for the business.
 
-With Odoo Sales, it is possible to instantly add an **Expiration Date** from the quotation or the
-sales order.
+Expiration date deadlines
+=========================
 
-.. image:: deadline/quotation_deadlines_1.png
+With Odoo *Sales*, it's possible to instantly add an expiration date in the :guilabel:`Expiration`
+field of a quotation or sales order.
+
+.. image:: deadline/quotation-deadlines-expiration-field.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to configure deadlines on Odoo Sales?
+   :alt: How to configure deadlines on Odoo Sales.
+
+Deadlines in quotation templates
+================================
+
+The Odoo *Sales* application also makes it possible to add a deadline to every quotation template,
+as well. Whenever a specific quotation template is used in a quote, its associated deadline is
+automatically applied.
+
+To add a deadline to a quotation template, navigate to :menuselection:`Sales app --> Configuration
+--> Quotation Templates`, and either select the desired quotation template to which a deadline
+should be added, or click :guilabel:`Create` to build a new quotation template from scratch.
+
+On the quotation template detail page, click the :guilabel:`Edit` button to edit the quotation.
 
-Use deadline in your quotation templates
-========================================
+Then, add a specific number of days to the :guilabel:`Quotation expires after` field, located
+beneath the quotation template name. The number of days represents how long the quotation will be
+valid for, before it expires. When done, click :guilabel:`Save`.
 
-It is also possible to add a deadline to every quotation template created. Whenever a specific
-quotation template is used in a quote, its associated deadline will be automatically applied. Be
-sure to check out our documentation about :doc:`quote_template` to excel in their use.
+Check out the documentation on :doc:`/applications/sales/sales/send_quotations/quote_template` to
+learn more.
```

Comment:
This would fit better as a **seealso**, like this: 
```suggestion
.. seealso::
  :doc:`/applications/sales/sales/send_quotations/quote_template`
```  

...however we already have one pointing to this doc so would remove it actually.
```suggestion
```